velox/docs/functions/spark/math.rst

```diff
@@ -14,12 +14,15 @@ Mathematical Functions
 .. spark:function:: ceil(x) -> [same as x]
 
     Returns ``x`` rounded up to the nearest integer.  
-    In spark only long, double, and decimal have ceil.
+    In Spark, Supported types are: BIGINT and DOUBLE.
 
-.. spark:function:: divide(x, y) -> [same as x]
+.. spark:function:: divide(x, y) -> double
 
-    Returns the results of dividing x by y. The types of x and y must be the same.
-    It always performs floating point division. Corresponds to spark's operator ``/``.
+    Returns the results of dividing x by y. It always performs floating point division.
```

Comment:
> It always performs

Drop "It always". 

`Performs floating point division.`

Reply:
Thanks, just fixed it.